lib/config/presets/internal/workarounds.ts

```diff
@@ -45,6 +24,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  breaking: {
```

Comment:
This should be similar to the "drop-in" approach. It shouldn't have its own packageRules